pep-9999.rst

```diff
@@ -0,0 +1,173 @@
+PEP: 9999
+Title: Format Specifier for Signed Zero
+Author: John Belmonte <john@neggie.net>
+Sponsor: Mark Dickinson
```

Comment:
Mark's email is missing.

Reply:
To note, there's a bit of an inconsistency between PEP-0001 and PEP-0012 here, as they both have header blocks but say somewhat different things in them (and your PEP template inside the `pep-0012` directory uses the PEP 1 header block). The former says only `<real name of sponsor>` for the sponsor field, and `<list of authors' real names and optionally, email addrs>` for the `Author` field, while the latter states `[Full Name <email at example.com>]` for both.

Probably something we should decide on and resolve, and maybe reduce the substantial duplication between the two (which besides being non-DRY leads to this sort of conflict and confusion) when it comes to header information. Want me to open an issue?

Reply:
You can see it in the python-dev archives at https://groups.google.com/g/dev-python/c/sdd97gYcf0Y dickinsm at gmail.

We should also get approval from him before merging this PR; I saw that he offered to sponsor on the bug, but he should get a chance to review the whole PEP too.